os/net/lwip/src/core/ipv4/ip_addr.c

```diff
@@ -53,9 +53,9 @@
  *
  */
 
-#include <net/lwip/opt.h>
-#include <net/lwip/ipv4/ip_addr.h>
 #include <net/lwip/netif.h>
+#include <net/lwip/debug.h>
+#include <arpa/inet.h>
```

Comment:
Why include the header file here, if the rest of the file does not use it? Isn't it better to include files where they are used directly?

Reply:
After adding Werror flag we were getting 'ntonl' related error to overcome that error we have added this header file.